README.md

```diff
@@ -256,28 +265,3 @@ tailwind classes (which are exported as static classes at the time of
 save/build). So even if the classes are on the list in tailwind.config.js, but
 they are not used by any html element at the time of running the app you cannot
 refer to them.
-
-### Social account login
```

Comment:
section not relevant any more as social login config happens entirely through env variables.